### ISSUE TITLE
Alt text and other accessibility goodies

### ISSUE THREAD

**Issue by dwaxe** [athensresearch/athens#723]
I imagine this is low-hanging fruit, but as I have never touched cljs in my life, I hope someone else can create an example PR that others like myself could follow. A good candidate to start with could perhaps be adding alt text to the [app toolbar buttons](https://github.com/athensresearch/athens/blob/c04067c588c1205987c49c404f09d64bcb248077/src/cljs/athens/views/app_toolbar.cljs#L98) or something similar.

Alt text could include keyboard shortcuts a la Slack 
<img width="124" alt="Screen Shot 2021-03-02 at 13 35 43" src="https://user-images.githubusercontent.com/3530333/109697437-0dd4b080-7b43-11eb-9dba-ecb777c6ec39.png">

**Comment by sid597** [comment]
Seems like a candidate for help pop up @julionav
